x/bank/Cargo.toml

```diff
@@ -9,7 +9,7 @@ gears = { path = "../../gears", features = ["cli", "export", "xmods"] }
 
 #newtypes
 bytes = { workspace = true }
-ibc-proto = { workspace = true } # TODO: should we import through gears via core-types?
+ibc-proto = { workspace = true }
```

Comment:
Why did you remove this?

Reply:
We had this discussion, or at least I think that we had. We need to export types in gears only if they being used more than once. So this comment is outdated.

Reply:
Maybe better example why we should omit re-export of structures is `ibc-rs` xmod which rely on different version of `ibc-proto` then `gears`